lib/stats/folder.go

```diff
@@ -15,6 +15,7 @@ import (
 type FolderStatistics struct {
 	LastFile LastFile  `json:"lastFile"`
 	LastScan time.Time `json:"lastScan"`
+	NextScan time.Time `json:"nextScan"`
```

Comment:
I see that this is saved into some LevelDB file. As the next scan time is calculated in any case after restarting, I guess that this shouldn't be saved in the first place. Do you know by chance where this serialization happens?